Rocket.Chat/Controllers/Chat/MessagesViewModel.swift

```diff
@@ -153,14 +153,14 @@ final class MessagesViewModel {
      */
     func item(for indexPath: IndexPath) -> AnyChatItem? {
         guard
-            indexPath.section < dataNormalized.count,
-            indexPath.row < dataNormalized[indexPath.section].elements.count
+            indexPath.section < dataSorted.count,
+            indexPath.row < dataSorted[indexPath.section].viewModels().count
```

Comment:
Let's request `viewModels()` just once and save it to get the right reference later?

Reply:
👍